### ISSUE TITLE
copyright information unclear

### ISSUE THREAD

**Issue by terceiro** [Mange/roadie#117]
Hi, I am packaging roadie for Debian as a dependency of Redmine, and the source code has two conflicts copyright statements. `README.md` says `Copyright (c) 2009-2015 Magnus Bergmark`, while `MIT-LICENSE` says `Copyright (c) 2009 Jim Neath / Purify`.

Do both copyright staments apply? Only one of them? Please clarify.

**Comment by Mange** [comment]
Sorry about that. The project's history lies in a fork, and we have a bit of a *[Ship of Theseus](https://en.wikipedia.org/wiki/Ship_of_Theseus)* kind of situation here in that all lines from the original software has been replaced since that fork. It's all been written by hand by me, or contributors working on the lines that were changed[1].

I'm having a hard time finding out about what happens with MIT license code that is completely rewritten.

---

[1]: To illustrate: Code structure and naming was completely different at one point, and most of the algorithms have been switched out or completely rewritten.
I can also scan for author information in the entire project in its current state, and I'd get this out:

```
› for f in **/*.rb; do git blame -C -- $f; done | sed "s/^[^(]*//" | sed "s/ [ 2].*\$//" | tr -d '(' | sort | uniq -c
     28 Aidan Feldman
      2 bbauer
   3773 Magnus Bergmark
      1 Pavel Zaitsev
      1 Roman Shterenzon
      6 Tyler Hunt
      4 Zhao Cai
```

In fact, if we look in *every* file in the repo, the only lines where Jim Neath is still the most recent author is the copyright notices:

```
› for f in **/*(.); do git blame -C -- $f | grep "(Jim Neath"; done
e2ebf23f (Jim Neath 2009-12-29 12:29:48 +0000  1) Copyright (c) 2009 Jim Neath / Purify
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  2) 
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  3) Permission is hereby granted, free of charge, to any person obtaining
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  4) a copy of this software and associated documentation files (the
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  5) "Software"), to deal in the Software without restriction, including
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  6) without limitation the rights to use, copy, modify, merge, publish,
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  7) distribute, sublicense, and/or sell copies of the Software, and to
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  8) permit persons to whom the Software is furnished to do so, subject to
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000  9) the following conditions:
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 10) 
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 11) The above copyright notice and this permission notice shall be
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 12) included in all copies or substantial portions of the Software.
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 13) 
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 14) THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 15) EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 16) MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 17) NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 18) LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 19) OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
^fb95d63 (Jim Neath 2009-12-29 12:05:42 +0000 20) WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
e74325dd readme.textile (Jim Neath       2009-12-29 20:33:16 +0000 457) 
e74325dd readme.textile (Jim Neath       2009-12-29 20:33:16 +0000 458) Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the ‘Software’), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
e74325dd readme.textile (Jim Neath       2009-12-29 20:33:16 +0000 459) 
e74325dd readme.textile (Jim Neath       2009-12-29 20:33:16 +0000 460) The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
e74325dd readme.textile (Jim Neath       2009-12-29 20:33:16 +0000 461) 
```

Basically, only the copyright notice in the current `README` and the `MIT-LICENSE` file was still created by him. As you can also see, the copyright notice in the `README` was discovered to be moved from an older `readme.textile` that no longer exists.

---

Now, I'm no lawyer, so I still think - despite this - that the safest bet to me is to leave him in and just update the license file with my name. What do you think about that?

**Comment by Mange** [comment]
I think this is fixed by b648a53d56464f2c4e8628e48d3e905581c21cd6. Please let me know if there are any other problems.

Does this need to be released as a new version, or is it enough that you have full LICENSE information here?

I'll make the same change in `roadie-rails` too.

**Comment by Mange** [comment]
`roadie-rails` have now been updated in Mange/roadie-rails@93c4904c28186e6752fa8d628694d1f0436b711a

**Comment by terceiro** [comment]
No I don't think you need to release a new version just for that; thanks.
